source/MRMesh/CMakeLists.txt

```diff
@@ -99,8 +99,6 @@ IF(NOT MR_EMSCRIPTEN)
 		IF(APPLE)
 			list(APPEND MRMESH_OPTIONAL_DEPENDENCIES OpenVDB)
 		ELSE()
-			target_include_directories(${PROJECT_NAME} PUBLIC ${MESHLIB_THIRDPARTY_LIB_DIR}/openvdb/openvdb/openvdb)
-			target_include_directories(${PROJECT_NAME} PUBLIC ${MESHLIB_THIRDPARTY_LIB_DIR}/openvdb/openvdb/openvdb/openvdb)
```

Comment:
dont need this anymore?

Reply:
yeah, because of this
https://github.com/MeshInspector/MeshLib/blob/fdba5212e57ae9f9d54a913b24c5ac10db8a8e6c/CMakeLists.txt#L55

and new thirdparty integration